Makefile

```diff
@@ -65,7 +65,7 @@ ENABLE_FEATURES += sse
 endif
 
 # Update Titan to latest master before build
-ifneq ($(UPDATE_TITAN), 0)
+ifeq ($(UPDATE_TITAN),1)
```

Comment:
This is horrible. Why update submodule during build time? This is an abuse of features. It can lead to unrepeatable build even on the same hash!

Reply:
agree, we should remove this update feature. it is horrible.

Reply:
used by test

Reply:
It is only set when `make titan_release`, which is used by Schrodinger. Without it, we can't do  daily bench/test for latest Titan.

Reply:
You can use dynamic link to achieve the goal.

Reply:
you mean to overload the dynamic link to rocksdb/titan when running? Let's discuss it offline so I can describe my demands more clearly.

Reply:
Could you use a custom script rather than the Makefile? It seems like something which is only correct to use for QA

Reply:
@nrc yes, but after discussion, we decide to remove this feature